data/devices/logitech-g-powerplay.device

```diff
@@ -3,6 +3,7 @@ Name=Logitech G Powerplay
 DeviceMatch=usb:046d:405f
 LedTypes=logo
 Driver=hidpp20
+DeviceType=mouse
```

Comment:
Create a new type, `Other`, and use it here. Powerplay is a mat.

Reply:
oh right, i missed that.